pyproject.toml

```diff
@@ -42,6 +42,10 @@ water_map = "asf_tools.water_map:main"
 calculate_hand = "asf_tools.hand.calculate:main"
 flood_map = "asf_tools.flood_map:main"
 
+[project.entry-points.hyp3]
```

Comment:
So what do these lines do exactly?

Reply:
These specify "hyp3" plugin entrypoints -- we commonly use the special console-scrpit entrypoints to register CLI entrypoints, but you can register functional entrypoints for *anything*. In this case, we specify a set of function entrypoints for hyp3. 
https://packaging.python.org/en/latest/specifications/entry-points/

Reply:
I've done it this way because we'd have to significantly change the CLI interface for `water_map` and `flood_map` otherwise

Reply:
used here:
https://github.com/ASFHyP3/asf-tools/pull/169/files#diff-2673b25a6a4398d07b383059f0daf23190e552c413fc6f2ebd8d11e3a118d511R15

Reply:
Seems reasonable, thanks for explaining that.